packages/services/src/services/KalturaPlayer/KalturaPlayer.js

```diff
@@ -234,17 +243,92 @@ class KalturaPlayerAPI {
           },
           // Ready callback is issued for this player:
           readyCallback: function(playerId) {
-            var kdp = document.getElementById(playerId);
-
-            kdp.addJsListener('playerPaused', function() {
-              fireEvent({ playerState: 1, kdp, mediaId });
-            });
-            kdp.addJsListener('playerPlayed', function() {
-              fireEvent({ playerState: 2, kdp, mediaId });
-            });
-            kdp.addJsListener('playerPlayEnd', function() {
-              fireEvent({ playerState: 3, kdp, mediaId });
-            });
+            const kdp = document.getElementById(playerId);
+
+            if (useIbmMetrics) {
+              let startingSeekTime = 0;
+
+              kdp.addJsListener('playerPaused.ibm', () => {
+                fireEvent({ playerState: 1, kdp, mediaId });
```

Comment:
Not sure if we wanted to include as part of this PR or perhaps a separate PR, if I recall we were discussing also creating standard event listeners that any application could also subscribe to if they wanted to monitor these behaviors (e.g. append their own custom tracking triggers), which would be triggered by these Kaltura listeners. I'd be fine with adding this in a separate PR though.